app/controllers/movies_controller.rb

```diff
@@ -6,7 +6,11 @@ class MoviesController < ApplicationController
   # GET /movies or /movies.json
   def index
     respond_to do |format|
-      @movies = Contexts::Movies::Queries::MovieQueries.new.all(page: params[:page])
+      @movies = if !params.key?(:search)
+                  Contexts::Movies::Queries::MovieQueries.new.all(page: params[:page])
+                else
+                  Contexts::Movies::Queries::MovieQueries.new.search_by(params[:search])
+                end
```

Comment:
let's swap conditions, then we can remove exclamation mark
```suggestion
      @movies = if params.key?(:search)
                  Contexts::Movies::Queries::MovieQueries.new.search_by(params[:search])
                else
                  Contexts::Movies::Queries::MovieQueries.new.all(page: params[:page])
                end
```